Hello Microverse: Setup

Shahid-Bagwan opened the pull request:
### Fix : " Kindly update the name of your feature branch to represent the general action that you performed in the branch "
Link to first PR : [https://github.com/Shahid-Bagwan/Hello-Microverse/pull/1](url)

Shahid-Bagwan committed:
created Feature branch & add index & css

created Feature branch & added index.html & style.css

Shahid-Bagwan committed:
updated style.css

Shahid-Bagwan committed:
add Readme

description of the project, why the project was create and what languages where used is written

V-Blaze commented on readme.md:
```diff
@@ -0,0 +1,288 @@
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
```
> Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

- [ ] You did great using the provided Microverse readme template. :+1: :+1: Looking at the readme file, it seems you have not customized it to your project. As the previous reviewer suggested and also According to Microverse the only optional section is the FAQ, you are meant to customize every other section in your readme file. :smile: :wave: I'll suggest you get the [template once more,](https://github.com/microverseinc/readme-template) make sure you do not remove any section. Then go through your readme and remove the placeholder logo and texts, and then fill the content of every section in the readme file :+1: :100: 

**THIS IS AN EXAMPLE OF HOW THE REAMDE SHOULD LOOK LIKE FOR THIS PROJECT. [LINK](https://github.com/enocol/HelloMicroverse/blob/newbranch/README.md)**

Shahid-Bagwan replied:
Update the readme file as you suggested

V-Blaze reviewed (changes requested):
## Hi @Shahid-Bagwan , 

# Almost there 🟡 

![1094a3e620ebea4cd6cf2d0aceaa8f52](https://user-images.githubusercontent.com/107632780/229645999-eb4dcb1d-5a0c-4bdd-a150-2ab5cee3f40d.gif)


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing some changes from the previous reviewers, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Meaningful pull request title and branch name
-  Linters setup correctly✔
- Correct Github flow💯 🥇 ✔

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Shahid-Bagwan committed:
Fix Readme

Filled details in readme file which were to understand the project. also addressed the issue by the code reviewer.

Shahid-Bagwan committed:
Little change in readme file

just removed a extra words, and solved typos.

Bantealem commented on readme.md:
```diff
@@ -0,0 +1,179 @@
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
```
You did a great change.  But as you don't have a log image you should remove this code.  Because it makes you `readme` unprofessional. Kindly remove it.

Shahid-Bagwan replied:
Done!

Bantealem commented on readme.md:
```diff
@@ -0,0 +1,179 @@
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Demo Link Comming soon](https://google.com)
+
```
As suggested by the previous reviewer, you need to customize the `readme` template. You are not required to deploy this project so the `deployment` section should be removed. Kindly fix it.

Shahid-Bagwan replied:
Done!

Bantealem commented on readme.md:
```diff
@@ -0,0 +1,179 @@
+
+## 📝 License <a name="license"></a>
+
+N/A
```
[OPTIONAL] Please also notice under the license section you should add a link that get into your project is licensed. You can  [find it here](https://github.com/microverseinc/readme-template/blob/master/MIT.md) and make sure the link points to that file using ./[repalce-wtih-file-name]. Kindly fix it.

Shahid-Bagwan replied:
Done!

Bantealem reviewed (changes requested):
Hi @Shahid-Bagwan 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![giphy](https://user-images.githubusercontent.com/74506933/153382064-ed760a8c-4f4a-48d6-b1f0-1aa9e5135097.gif)




 ## Highlights
:heavy_check_mark:  No Linters Error
:heavy_check_mark:  Correct Github flow.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Shahid-Bagwan committed:
Add License file & Update readme

added license file & made changes which were suggested by code reviewer.

Shahid-Bagwan committed:
Update readme

removed an unwanted line from the file.

KanzaTahreem commented on readme.md:
```diff
@@ -0,0 +1,161 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```
- [OPTIONAL] I think it would be better to remove the Live Demo from table of contents for maintaining a consitency and a professional look. What do you think? ✨💪

KanzaTahreem reviewed (approved):
# Approved 🥳

Hi @Shahid-Bagwan 🙋‍♀️

Great work implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**

<img src="https://media4.giphy.com/media/cFkiFMDg3iFoI/giphy.gif"/>

## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅`test.md` is ignored, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Your README looks professional, well done!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
